### PR TITLE
Flow13 and workaround

### PR DESCRIPTION
**Pull request by vrindle**

### CODE REVIEW

File: process-cmds.sh [hunk 1]
```diff
@@ -202,98 +202,98 @@ inspect-system-ovs-ovn() {
...
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/show\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: sudo ovs-appctl dpctl/dump-flows --names -m" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/dump-flows --names -m\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/dump-flows --names -m\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "" >> "${HWOL_VALIDATION_FILENAME}"
   echo "## Kernel TC filter Information" >> "${HWOL_VALIDATION_FILENAME}"
   # Get PF from br-ex
   PF_NAME=`kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ovs-vsctl list-ports br-ex | grep -v patch-br-ex\""`
```
Comment (wizhaoredhat) [inline process-cmds.sh:237]: Does this command work? Without the cat?

File: variables.sh [hunk 2]
```diff
@@ -24,6 +24,7 @@ FT_MC_CO_SERVER_LABEL=${FT_MC_CO_SERVER_LABEL:-submariner.io/gateway=true}
 NET_ATTACH_DEF_NAME=${NET_ATTACH_DEF_NAME:-ftnetattach}
 SRIOV_RESOURCE_NAME=${SRIOV_RESOURCE_NAME:-openshift.io/mlnx_bf}
 TEST_IMAGE=${TEST_IMAGE:-quay.io/wizhao/ft-base-image:0.9}
+IPERF3_SERVER_NAME="iperf3-server"
```
Comment (wizhaoredhat) [inline variables.sh:27]: We can be more specific here E.g.
`EXTERNAL_IPERF_SERVER_POD_NAME="external-iperf-server"`

File: process-cmds.sh (hunk 1 above)
Comment (vrindle) [reply process-cmds.sh:237]: it does not work without the cat. It does work with piping to cat.

File: process-cmds.sh [hunk 3]
```diff
@@ -202,98 +202,98 @@ inspect-system-ovs-ovn() {
...
   echo "### Command: ethtool -i ovn-k8s-mp0_0" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0_0\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0_0\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: cat /proc/net/nf_conntrack" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /proc/net/nf_conntrack\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /proc/net/nf_conntrack\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "" >> "${HWOL_VALIDATION_FILENAME}"
   echo "## OvS Information" >> "${HWOL_VALIDATION_FILENAME}"
   # Workaround: https://bugzilla.redhat.com/show_bug.cgi?id=1803920
   OPENVSWITCHPID=`kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /var/run/openvswitch/ovs-vswitchd.pid\""`
```
Comment (wizhaoredhat) [inline process-cmds.sh:227]: One more up here.

Comment (vrindle) [reply process-cmds.sh:227]: So this also works with and without piping to cat.

File: process-cmds.sh (hunk 1 above)
Comment (wizhaoredhat) [reply process-cmds.sh:237]: Great, that means we can keep it as is.

File: process-cmds.sh (hunk 3 above)
Comment (wizhaoredhat) [reply process-cmds.sh:227]: Can keep as is.

File: variables.sh (hunk 2 above)
Comment (wizhaoredhat) [reply variables.sh]: Can we refactor IPERF3_SERVER_NAME -> EXTERNAL_IPERF_SERVER_POD_NAME ?

**Pull request merged**